content/maxwell2_steady_state/SphericalDepression_code.rst

```diff
@@ -0,0 +1,8 @@
+.. _effecttopo_code:
```

Comment:
do you guys want this included in a table of contents somewhere? or only a link to it?